### PR TITLE
Print library versions the proper way

### PR DESCRIPTION
**Pull request by hynek**
Reporting the versions from inside is nicer and will also help us if people post their non-tox test failures.

Makes the header look like this for me:

```
platform darwin -- Python 2.7.10, pytest-2.9.0, py-1.4.31, pluggy-0.3.1
OpenSSL: OpenSSL 1.0.2g  1 Mar 2016
cryptography: 1.3.dev1
rootdir: /Users/hynek/Projects/pyopenssl, inifile: setup.cfg
collected 459 items
```

### CODE REVIEW

**Comment by reaperhulk** [comment]
You should be able to put conftest in the `tests` dir (and then you won't need to modify the MANIFEST.in)

**Comment by hynek** [comment]
Yeah there are various opinions on where to keep it, I like it more in tests too.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include             LICENSE MANIFEST.in *.rst tox.ini docs-requirements.txt .coveragerc
+include             LICENSE MANIFEST.in *.rst tox.ini docs-requirements.txt .coveragerc conftest.py
```
Comment (reaperhulk) [inline MANIFEST.in]: Shouldn't need this addition any more as the recursive-include on tests will catch it.

**Comment by reaperhulk** [comment]
Whoops, was looking at an out of date diff somehow.

**Pull request merged**